pkg/fakerp/dns.go

```diff
@@ -4,7 +4,7 @@ import (
 	"context"
 	"strings"
 
-	azdns "github.com/Azure/azure-sdk-for-go/services/dns/mgmt/2017-10-01/dns"
+	azuredns "github.com/Azure/azure-sdk-for-go/services/dns/mgmt/2017-10-01/dns"
```

Comment:
nit: necessary change 
same with `zc` -> `zoneClient`

Reply:
`azdns` to `azuredns`, `dnsManager.zc` to `dnsManager.zonesClient` and `dnsManager.rsc` to `dnsManager.recordSetsClient` were all changed to enhance readability. I feel the code is slightly more readable now.